### ISSUE TITLE
Module '../polymer-analyzer/lib/index"' has no exported member 'FsUrlLoader'. Did you mean 'FSUrlLoader'

### ISSUE THREAD

**Issue by TvrtkoM** [Polymer/tools#200]
I'm getting the error in title when trying to run the following code:

```
import {PolymerProject} from 'polymer-build';

const project = new PolymerProject({
  root: "src/html-modules"
});
```
package.json deps:
```
  "dependencies": {
    "polymer-build": "^2.5.0",
    "typescript": "^2.8.3"
  }
```

```
-> % ./node_modules/.bin/tsc -v     
Version 2.8.3
```

tsconfig:
```
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "es2016",
      "es2017",
      "esnext.asynciterable"
    ],
    "module": "commonjs",
    "rootDir": "src",
    "outDir": "lib"
  },
  "include": [
    "src/**/*"
  ]
}
```

What am I doing wrong?

**Comment by TvrtkoM** [comment]
I fixed it with `npm i --save polymer-build@3.0.0-pre.13`. Strange dependency with analyzer on version 2.5.0 of polymer-build

**Comment by TimvdLippe** [comment]
I am unable to reproduce this issue, so I suppose this has been fixed along the way.
